app/Http/Controllers/Web/ProfileSubscriptionsController.php

```diff
@@ -24,7 +30,10 @@ public function __construct()
      */
     public function edit()
     {
-        return view('profiles.subscriptions.edit', ['user' => auth()->guard('web')->user()]);
+        return view('profiles.subscriptions.edit', [
+            'user' => auth()->guard('web')->user(),
+            'intended' => session()->pull('url.intended') ?: '/',
```

Comment:
what does the 'intended variable' here add? I can make an assumption but I haven't seen it used thus far!

Reply:
Yeah sorry, I should have clarified! This is for the "Skip" button. Since this is the last onboarding form page, that button would need to lead to the user's intended destination before being directed to authenticate (or defaults to the home page).
https://github.com/DoSomething/northstar/blob/d2358570fa4281443501cba81dc3daba63e4ac0c/resources/views/profiles/subscriptions/edit.blade.php#L63

We pull it from the session and pass it along to the view. (I'm also realizing now that this could be slightly simplified!)

```suggestion
            'intended' => session()->pull('url.intended', '/'),
```